beerocks/bwl/dummy/mon_wlan_hal_dummy.h

```diff
@@ -52,10 +52,10 @@ class mon_wlan_hal_dwpal : public base_wlan_hal_dwpal, public mon_wlan_hal {
 
     // Private data-members:
 private:
-    std::shared_ptr<char> m_temp_dwpal_value;
+    std::shared_ptr<char> m_temp_dummy_value;
```

Comment:
I don't think this is used anywhere.

Reply:
It is just copy-paste from dwpal bwl without much cleanups other than making everything as stub and having this compile and get to ~operational with controller + agent.
I don't think cleanups are reqyured in the dummy files since we plan to delete this after bringing up dwpal + hwsim.

Reply:
OK, so I'll lighten up my review.